### ISSUE TITLE
Comment contains "#unknown are still not solved"

### ISSUE THREAD

**Issue by php-coder** [yegor256/0pdd#234]
When 0pdd adds a comment to a parent issue about number of not solved puzzles, the comment may contain `#unknown are still not solved` that is, as I suspect, a bug.

Example:
> 19 puzzles #839, #841, #842, #846, #847, #849, #851, #852, #853, #854, #855, #856, #859, #861, #863, #865, #866, #867, #unknown are still not solved; solved: #833, #834, #835, #836, #838, #840, #843, #844, #848, #860, #864.

Seen there: https://github.com/php-coder/mystamps/issues/695#issuecomment-405372820

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
@php-coder[/z](https://www.0crat.com/u/php-coder) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3T46CUJJ) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
Job [#234](https://github.com/yegor256/0pdd/issues/234) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Thanks for your contribution, @php-coder[/z](https://www.0crat.com/u/php-coder)! If you would be a member of [the project](https://www.0crat.com/p/C3T46CUJJ), you would now earn +15 reputation points, as explained in [§29](http://www.zerocracy.com/policy.html#29). You can join and apply to it, see [§2](http://www.zerocracy.com/policy.html#2).

**Comment by 0crat** [comment]
The job [#234](https://github.com/yegor256/0pdd/issues/234) assigned to @snugghash[/z](https://www.0crat.com/u/snugghash), here is [why](https://www.0crat.com/footprint/C3T46CUJJ/2610536459); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by snugghash** [comment]
The issue seems to be that there's no testing done for this sort of runtime behaviour in: [test_notification_on_one_solved_update](https://github.com/yegor256/0pdd/blob/master/test/test_diff.rb#L128) doesn't catch it.

Is this even testable? The bug seems to be that the correct link is formed, but the text is bad. 

"Unknown" issues sem to be only referenced in [puzzzles.rb, line 71](https://github.com/yegor256/0pdd/blob/745073b29a952e6023fa2306a0a53b6817356e8c/objects/puzzles.rb#L71)

On paper, it seems like we can write a `test_notrification_on_many_solved_including_parent` to narrow this down.

**Comment by 0crat** [comment]
@snugghash[/z](https://www.0crat.com/u/snugghash) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by snugghash** [comment]
@php-coder Please review and close.

**Comment by 0crat** [comment]
The user @snugghash[/z](https://www.0crat.com/u/snugghash) resigned from [#234](https://github.com/yegor256/0pdd/issues/234), please stop working. Reason for job resignation: It is older than 10 days, see [§8](http://www.zerocracy.com/policy.html#8)

**Comment by 0crat** [comment]
Resigned on delay, see [§8](http://www.zerocracy.com/policy.html#8): -30 point(s) just awarded to @snugghash[/z](https://www.0crat.com/u/snugghash)

**Comment by snugghash** [comment]
@yegor256 ARC, please assign me back to the issue so I can put this in wait mode.

**Comment by php-coder** [comment]
> @php-coder Please review and close.

@snugghash Don't wait for me. I test it only when it will be in published (merged and deployed). Also, in order to test it, I should close one of my tasks, probably, that also takes time (and I'm not sure that I'll find it).

**Comment by snugghash** [comment]
@php-coder  I understand, in the end there are a lot of dependencies. For that specific PR, I'd still be be waiting on a (manually assigned?) review from REV role-person. Is that acceptable?

**Comment by ypshenychka** [comment]
@yegor256 Please assign this issue back to @snugghash. Thanks.

**Comment by yegor256** [comment]
@0crat assign @snugghash

**Comment by 0crat** [comment]
> @0crat assign @snugghash  ([here](https://github.com/yegor256/0pdd/issues/234#issuecomment-435653505))

@yegor256 The job [#234](https://github.com/yegor256/0pdd/issues/234) assigned to @snugghash[/z](https://www.0crat.com/u/snugghash), here is [why](https://www.0crat.com/footprint/C3T46CUJJ/b776e65d-e830-47a7-8b8a-afa13e4876b2); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); we should be aware that @snugghash[/z](https://www.0crat.com/u/snugghash) is on vacation; this ticket may be delayed; there will be a monetary reward for this job

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The user @snugghash[/z](https://www.0crat.com/u/snugghash) resigned from [#234](https://github.com/yegor256/0pdd/issues/234), please stop working. Reason for job resignation: It is older than 10 days, see [§8](http://www.zerocracy.com/policy.html#8)

**Comment by 0crat** [comment]
Resigned on delay, see [§8](http://www.zerocracy.com/policy.html#8): -30 point(s) just awarded to @snugghash[/z](https://www.0crat.com/u/snugghash)

**Comment by php-coder** [comment]
It seems like the issue is fixed now: https://github.com/php-coder/mystamps/issues/671#issuecomment-443468450

Thanks!

**Comment by 0crat** [comment]
Job `gh:yegor256/0pdd#234` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#234](https://github.com/yegor256/0pdd/issues/234) is now out of scope

**Comment by php-coder** [comment]
Still here: https://github.com/php-coder/mystamps/issues/695#issuecomment-447992236

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
@php-coder[/z](https://www.0crat.com/u/php-coder) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3T46CUJJ) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by yegor256** [comment]
@php-coder please, open a new ticket. Thanks!

**Comment by 0pdd** [comment]
@php-coder the puzzle [#386](https://github.com/yegor256/0pdd/issues/386) is still not solved.

**Comment by 0pdd** [comment]
@php-coder the only puzzle [#386](https://github.com/yegor256/0pdd/issues/386) is solved here.

**Comment by 0pdd** [comment]
@php-coder the only puzzle [#386](https://github.com/yegor256/0pdd/issues/386) is solved here.
